docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+  mongo:
+    image: mongo:3.4
+    networks:
+      default:
+        aliases:
+          - mongo
+    ports:
+      - "27017:27017"
+  backend:
+    build:
+      context: .
+      dockerfile: dockerbuild/Dockerfile-backend
+    image: mitdbg/modeldb-backend:latest
+    networks:
+      default:
+        aliases:
+          - backend
+    ports:
+      - "6543:6543"
+    command: ["mongo"]
+  frontend:
+    build:
+      context: .
+      dockerfile: dockerbuild/Dockerfile-frontend
+    image: mitdbg/modeldb-frontend:latest
+    networks:
+      default:
+        aliases:
+          - frontend
+    ports:
+      - "80:3000"
```

Comment:
Also, unless we plan to start the frontend on 80 for the non-docker setup as well, can we stick with 3000 for consistency?